PACKAGE.md

```diff
@@ -30,11 +30,11 @@ from membrane import configure
 
 # Configuration
 configure(
+    active=True
```

Comment:
I think this might be worse actually. My point was more to either not have a parameter or show the developer how to make this conditional to MEMBRANE_BACKEND being defined (as you've done in the test). There's actually three cases:

1) MEMBRANE_BACKEND is not defined (so require_login is false)
2) MEMBRANE_BACKEND is defined but deployer wants to disable it (require_login is false)
3) MEMBRANE_BACKEND is defined and deployer wants it enabled (require_login is true)

probably worth discussing voice to dig into this subject.